README.md

```diff
@@ -1,116 +1,74 @@
-# Auth0 - Ruby
+# ruby-auth0
+
+Ruby API client for the [Auth0](https://auth0.com) platform.
 
 [![CircleCI](https://img.shields.io/circleci/project/github/auth0/ruby-auth0/master.svg)](https://circleci.com/gh/auth0/ruby-auth0)
 [![Gem Version](https://badge.fury.io/rb/auth0.svg)](http://badge.fury.io/rb/auth0)
 [![codecov](https://codecov.io/gh/auth0/ruby-auth0/branch/master/graph/badge.svg)](https://codecov.io/gh/auth0/ruby-auth0)
 [![Yard Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/auth0/ruby-auth0/master/frames)
 [![MIT licensed](https://img.shields.io/dub/l/vibe-d.svg?style=flat)](https://github.com/auth0/ruby-auth0/blob/master/LICENSE)
-[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fauth0%2Fruby-auth0.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fauth0%2Fruby-auth0?ref=badge_shield)
 
-Ruby API client for the [Auth0](https://auth0.com) platform.
+:books: [Documentation](#documentation) - :rocket: [Getting Started](#getting-started) - :computer: [API Reference](https://www.rubydoc.info/gems/auth0) - :speech_balloon: [Feedback](#feedback)
```

Comment:
As I had some troubles with this for python, does rubydoc support these emoji's ? It looks like it uses the readme as the package's homepage: https://www.rubydoc.info/gems/auth0

I solved it in python as seen here: https://github.com/auth0/auth0-python/pull/448

Reply:
It's a good question, I don't know. I will use your method from Python anyway as a precaution 👍🏻